.promu.yml

```diff
@@ -0,0 +1,44 @@
+repository:
+    path: github.com/cloudflare/alertmanager2es
+build:
+    binaries:
+        - name: alertmanager2es
+          path: .
+    flags: -a -tags netgo
+    ldflags: |
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Version={{.Version}}
```

Comment:
None of these would have any effect currently, though I'd welcome changes to make use of the `version` package.

Currently, only `main.revision` is set during compile time.